packages/tool-studio/src/ui/app/styles.css

```
@@ -53,83 +71,160 @@
+  --sb-code-function: #78aee8;
+  --sb-code-string: #9fbd73;
+  --sb-code-number: #bd93d8;
+  --sb-code-literal: #df7d68;
+  --sb-code-comment: #56606a;
 }
 
 @layer base {
   * {
-    @apply border-border outline-ring/50;
+    @apply border-border outline-ring;
     box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)